src/components/creator.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <el-card shadow="hover">
+    <el-row :gutter="40">
+      <el-col :span="5">
+        <img
+          v-bind:src="'https://q.trap.jp/api/1.0/public/icon/' + traQID"
+          width="40"
+        />
+      </el-col>
+      <el-col :span="6" class="text">@{{ traQID }} </el-col>
+    </el-row>
+  </el-card>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
```

Comment:
あんまりきちんときめてはないんだけど、component の属性に `name` がほしいかも
+ファイル名も含め CreatorElement のほうがいいかも
```suggestion
export default defineComponent({
  name: 'CreatorElement'
```